client/.env

```diff
@@ -0,0 +1 @@
+SKIP_PREFLIGHT_CHECK=true
```

Comment:
I can't find where you're using this variable.

Reply:
I was getting an error from the Create-React-App because I had an eslintrc.json both in the root directory (for the server) and the client directory. Adding this line to the .env makes create react app ignore the error and start the client anyway. I'm not sure this is the best way to fix it, but I didn't feel like debugging last night.